cc/testing/stochastic_tester_test.cc

```diff
@@ -131,7 +131,7 @@ template <typename T>
 class CountNoDpError : public Count<T> {
  public:
   explicit CountNoDpError(double epsilon)
-      : Count<T>(epsilon, LaplaceMechanism::Builder()
+      : Count<T>(epsilon, 1e-5, LaplaceMechanism::Builder()
```

Comment:
Same here, this should also work with delta = 0

Reply:
Fixed in the next commit.